### DISCUSSION TITLE
No option for allowing each file to be an audiobook.

### DISCUSSION THREAD

**Discussion by Dr-Ari-Gami** [PaulWoitaschek/Voice#1838] [Ideas]
### Overview

I have a well organized root audiobook folder. I sort each of my audiobooks by author, and then by book. Unfortunately, I've run into a problem where different books by the same author are considered to be the same book. See screenshot below:

![Screenshot_20230427-223220](https://user-images.githubusercontent.com/33817588/235062637-188176b3-b1ac-401e-baf4-a6f694e4cc61.png)

Even with the `Root Audiobook Folder` option selected, it still combinds every book by one author into one. 

![Screenshot_20230427-223423](https://user-images.githubusercontent.com/33817588/235063062-61a2735b-68ba-44d3-8e54-1fa24b334a6d.png)

Can an option be implemented that will allow a root folder to be imported, and not combind multiple audio files into one?

### How to reproduce

Important a folder with multiple subdirectory trees. 

For instance:
```
Library
---- James S. A. Corey
-------- Abaddon's Gate
------------ Abaddon's Gate James S. A. Corey.m4b
-------- Caliban's War
------------ Caliban's War - James S. A. Corey.m4b
-------- Leviathan Wakes
------------ Leviathan Wakes James S. A. Corey.m4b
```

### Version

7.0.7

### Android Version

13

### Device Model

Google Pixel 6

**Comment by ghost** [comment]
The easiest way to fix this is to organize your audiobook folder differently for Voice. 

Either move all the book folders up one folder into the root folder and out of the author folder or put all your files into one audiobook folder without any subfolders.

**Comment by Dr-Ari-Gami** [comment]
I know, I'm just hoping I don't have to. 

I import my entire calibre library, audiobooks included, and would rather stick with that organization if possible.

**Comment by ghost** [comment]
Another solution would be to add every author folder separately as a root audiobook folder. Depending on the number of authors this could be a very annoying process.
